src/fusor/examples/bcr_abl1.json

```diff
@@ -1,111 +1,131 @@
 {
   "type": "CategoricalFusion",
-  "structural_elements": [
-    {
-      "type": "TranscriptSegmentElement",
-      "transcript": "refseq:NM_004327.3",
-      "gene_descriptor": {
-        "type": "GeneDescriptor",
-        "id": "normalize.gene:BCR",
-        "gene_id": "hgnc:1014",
-        "label": "BCR"
+  "structure": {
+    "type": "Adjacency",
+    "adjoinedSequences": [{
+      "type": "SequenceLocation",
+      "sequenceReference": {
+        "id": "GRCh38:chr22",
+        "type": "SequenceReference",
+        "refgetAccession": "SQ.7B7SHsmchAR0dFcDCuSFjJAo7tX87krQ",
+        "residueAlphabet": "na"
       },
-      "element_genomic_end": {
-        "id": "fusor.location_descriptor:NC_000022.11",
-        "type": "LocationDescriptor",
-        "label": "NC_000022.11",
-        "location": {
-          "type": "SequenceLocation",
-          "sequence_id": "refseq:NC_000022.11",
-          "interval": {
-            "type": "SequenceInterval",
-            "start": {
-              "type": "Number",
-              "value": 23253980
-            },
-            "end": {
-              "type": "Number",
-              "value": 23253981
+      "end": 23290413,
+      "extensions": [
+        {
+          "name": "NM_004327.4:e._14",
```

Comment:
This extension to the genomic coordinate shows the representation of the transcript segment boundary in terms of the VICC exon representation.